src/dmd/module.h

```diff
@@ -75,6 +82,7 @@ class Module : public Package
 
 
     const char *arg;    // original argument name
+    Array<QualifiedID> importNames; // a list of names that can be used to import this module
```

Comment:
s/a list of names/names/

as it's obviously an array of names, not a list of names